tests/test_gitlab.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import pytest
+from functools import wraps
+from utils import (
+    databricks_cli,
+    generated_project_dir,
+    parametrize_by_cloud,
+)
+
+
+@pytest.mark.parametrize("cicd_platform", ["gitlab"])
+@pytest.mark.parametrize(
+    "setup_cicd_and_project,include_feature_store,include_mlflow_recipes,include_models_in_unity_catalog",
+    [        
+        ("CICD_and_Project", "no", "no", "no"),
+        ("CICD_and_Project", "no", "no", "yes"),
+        ("CICD_and_Project", "no", "yes", "no"),
+        ("CICD_and_Project", "yes", "no", "no"),
+        ("CICD_and_Project", "yes", "no", "yes"),
+        ("CICD_Only", "no", "no", "no")
```

Comment:
What about the other `CICD_only` options?

Reply:
I copied all the options from github tests.
I think for **CICD_Only** the other options are not relevant (include_feature_store, include_mlflow_recipes, include_models_in_unity_catalog are only relevant for the **Project** part!) so we leave all set to 'no'

Reply:
Sure that's fine then, thanks!

Reply:
on this file I've now added the test for all clouds